pkg/data/data_store.go

```diff
@@ -12,7 +10,7 @@ import (
 const collection = "data"
 
 type IDataStore interface {
-	Get(order string, skip, take int, startDate, finishDate time.Time)
+	Get(query *entities.Query)
 	Save(data entities.Data)
 	Delete(deviceID string)
 }
```

Comment:
Shouldn't the interfaces have the return types too? :point_down: 

```Go
type IDataStore interface {
	Get(query *entities.Query) ([]entities.Data, error)
	Save(data entities.Data) error
	Delete(deviceID string) error
}
```

Reply:
Done